.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,14 +6,14 @@ A brief description of the change being made with this pull request.
 
 What inspired you to submit this pull request?
 
-### Review checklist
-
-- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
-- [ ] Feature or bugfix has tests
-- [ ] Git history is clean
-- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)
-- [ ] If PR adds a configuration option, it has been added to the configuration file.
-
 ### Additional Notes
 
 Anything else we should know when reviewing?
+
+### Review checklist (to be filled by reviewers)
+
+- [ ] PR title is written as a CHANGELOG entry [(see why)](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title)
+- [ ] PR has the `changelog/` and `integration/` labels attached
```

Comment:
Let's add this link https://keepachangelog.com/en/1.0.0/ maybe?

Reply:
Not really, mentioned labels are part of our release process, they're simply mandatory